docs/project/development-process/koji-initial-setup.md

```diff
@@ -64,6 +64,11 @@ In some cases (with older versions of koji), we've found that the `cert` directi
 
 ## Get more permissions
 After your first connection, your user gets added by koji to its database. Now you need permissions to be able to tag builds. Ask an administrator for the `build` permission.
+Permissions can be double checked by user using:
+```
+koji list-users # Check if $USER is listed
+koji list-permissions --user=$USER # Adapt $USER if needed
+```
```

Comment:
```suggestion

You can verify the permissions for a given user `$USER` with:
\```
koji list-users # Check if $USER is listed
koji list-permissions --user=$USER # Adapt $USER if needed
\```
```